migration.properties

```diff
@@ -20,29 +20,88 @@
 # By default no tags are created.
 #
 # Example for all baselines
-# rtc.baseline.include=^(.*)$
+rtc.baseline.include=^(.*)$
 #
 #rtc.baseline.include=
 
+# Ignore changesets that do not result in changes to any (non-ignored) files.
+# Making a source code change that has no changes is usually a mistake and clients usually
+# prevent users from doing this by accident.
+# With RTC, it's possible to deliver changesets that have no changes, or (if the original
+# author was "particularly inventive") which only change ignored files.
+# This scenario results in an empty commit for git, and by default git will prevent empty
+# commits from being added, so by default the migration tool skips these.
+# You can either ignore these commits (the default) or, if this is set to "true", you
+# can allow these empty commits to be added to the git history to preserve the empty change
+# as-is.
+#
+# git.allow.empty.commits=false
+git.allow.empty.commits=true
```

Comment:
I found it useful to ensure a 1-to-1 relationship between RTC and git commits during migration.
Git let's you rewrite history to be neater if you want, so that's what I did - I aimed to migrate RTC to git "as is" and then tidy up once the data was safely in git.

i.e. I'd recommend changing the default.

Reply:
Ok for me, since you have used it more lately 😎